code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -84,6 +84,15 @@
 	burntime = 20
 	// TODO:  Special ash subtype that looks like charred chair legs
 
+/obj/structure/stool/bed/chair/wood/attack_animal(mob/living/simple_animal/user)

```

Comment:
These are currently the only major instances of this in the code right now. A refactoring of this would make sense for the future, so I'll add a comment in, but for the sense of just getting this issue fixed for now, I think this is fine, no?